### PR TITLE
tests: add bare-metal log files to ELK

### PR DESCRIPTION
**Pull request by erikkn**
This PR will call the script that sends log files to S3 for the bare-metal tests.

### CODE REVIEW

**Comment by erikkn** [comment]
@cpanato

**Comment by cpanato** [comment]
retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -503,6 +503,11 @@ def runRSpecTestBareMetal(testFilePath, credentials) {
         reportStatusToGithub((err == null) ? 'success' : 'failure', testFilePath, originalCommitId)
         archiveArtifacts allowEmptyArchive: true, artifacts: 'build/**/logs/**'
         cleanWs notFailBuild: true
+        withCredentials(creds) {
```
Comment (cpanato) [inline Jenkinsfile]: you need to use `credentials` instead of `creds`

**Comment by cpanato** [comment]
@erikkn  after merge this, you need to cherry pick to track 1

**Pull request merged**